src/dbnode/encoding/null.go

```diff
@@ -42,8 +42,8 @@ func NewNullEncoder() Encoder {
 func (e *nullEncoder) Encode(dp ts.Datapoint, timeUnit xtime.Unit, annotation ts.Annotation) error {
 	return nil
 }
-func (e *nullEncoder) Stream() xio.SegmentReader {
-	return xio.NewSegmentReader(ts.Segment{})
+func (e *nullEncoder) Stream() (xio.SegmentReader, bool) {
+	return xio.NewSegmentReader(ts.Segment{}), false
```

Comment:
Either `xio.NewSegmentReader(ts.Segment{}), true` or `nil, false`

Reply:
actually gonna return an empty reader and also false (to indicate its empty)